addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -45,6 +46,7 @@ odoo.define('point_of_sale.Chrome', function(require) {
             useListener('show-notification', this._onShowNotification);
             useListener('close-notification', this._onCloseNotification);
             useListener('connect-to-proxy', this.connect_to_proxy);
+            useListener('print-sale-details', this.printSaleDetailsReport);
```

Comment:
So you suggest we instead keep the `SaleDetailsButton` component and insert it in two different places, one at header, as was before, and the other at the closingPosPopup, right?
When you say "accept some new props to replace the image/classes needed for the different situations" can you provide some examples because I don't clearly see the whole scope of it? Thankss! :+1:

Reply:
Hi @trj-odoo, I have adapted the `SaleDetailsButton` component like you suggested, and removed the trigger all the way to `Chrome` component, you can have a look and let me know what you think.

Reply:
a lot better :+1: